tests/nrf5340_audio/sw_codec_lc3/readme.txt

```diff
@@ -0,0 +1,6 @@
+# LC3 test
+This test can only be run on nRF5340 Audio application because the `qemu_cortex_m3` target does not supporting FPU.
+
+To run the test on target, run the following command, with the the COM port (here, `/dev/ttyACM2`) set to the APP port of your development kit:
+
+/zephyr/scripts/sanitycheck --device-testing --device-serial /dev/ttyACM2 --platform nrf5340dk_nrf5340_cpuapp -T . -v
```

Comment:
This is not a showstopper, but please change in the future this command for:
```
zephyr/scripts/twister --device-testing --device-serial /dev/ttyACM2 --platform nrf5340dk_nrf5340_cpuapp -T nrf/tests/nrf5340_audio/sw_codec_lc3 -v
```
`sanitycheck` was replaced by `twister` almost 1,5 years ago.

BTW if you are interested in testing program which requires FPU access on QEMU, then you can consider use this platform: `mps2_an521_remote`